### DISCUSSION TITLE
Header and Footer blocking the screen

### DISCUSSION THREAD

**Discussion by Pathi-rao** [OpenMined/courses#356] [Feedback] [answered]
Hi,
I would like to suggest an improvement for the UI while watching the course videos.

### Problem: 
While watching the videos, part of the video is getting blocked by the header(the dashboard part) and footer(the feedback part). One has to scroll up or down in order to see the blocked part of the video.  We can avoid this by going full screen but for someone who's coding along(like most of us do), it gets very annoying going full screen on and off every time.

### Suggestion:
When the user's cursor is inactive for few seconds, the header and footer will disappear, revealing the entire screen and when the user moves the cursor, they pop up back or maybe something like a hover effect.

**Comment by znreza** [comment, marked as answer]
Hi @Pathi-rao thanks for your feedback! @yemikifouly could you please convey this suggestion to the dev team? :)
